AS 3.3 Build errors

##### Detailed steps causing the problem:
- Install/Upgrade Android Studio to version 3.3

##### Actual behavior after performing these steps:
Build works not more. Maybe from this error:
"D:\git\cgeo\main\build\intermediates\incremental\mergeBasicDebugResources\merged.dir\values\values.xml:3609: AAPT: error: resource android:attr/preserveIconSpacing is private.
error: failed linking references.
	at com.android.builder.internal.aapt.v2.Aapt2Exception$Companion.create(Aapt2Exception.kt:45)
...
Caused by: com.android.builder.internal.aapt.v2.Aapt2Exception: Android resource linking failed"

##### Expected behavior after performing these steps:
Gradle sync/Build with warnings as before

##### Version of c:geo used:
current master branch

##### Is the problem reproducible for you?
Yes

##### System information:
-

##### Other comments and remarks:
I think this point in GradleSync is the reason:
"WARNING: The following project options are deprecated and have been removed: 
android.enableAapt2
This property has no effect, AAPT2 is now always used.
Affected Modules: cgeo-contacts, main, mapswithme-api"

Maybe also this point:
"WARNING: Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'.
It will be removed at the end of 2018. For more information see: http://d.android.com/r/tools/update-dependency-configurations.html
Affected Modules: main"

--- a/build.gradle
+++ b/build.gradle
@@ -75,7 +75,7 @@ buildscript {
         classpath 'com.github.ben-manes:gradle-versions-plugin:0.17.0'
 
         // Android gradle plugin
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.3.1'
 
         // un-mocking of portable Android classes
         classpath 'de.mobilej.unmock:UnMockPlugin:0.6.4'

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

--- a/main/build.gradle
+++ b/main/build.gradle
@@ -243,7 +243,7 @@ android {
 
 dependencies {
     // AndroidAnnotations, https://github.com/excilys/androidannotations/wiki/building-project-gradle
-    def androidAnnotationsVersion = '4.5.2'
+    def androidAnnotationsVersion = '4.6.0'
     annotationProcessor "org.androidannotations:androidannotations:$androidAnnotationsVersion"
     implementation "org.androidannotations:androidannotations-api:$androidAnnotationsVersion"
 

